testing/e2e/init.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Configure git
+git config --global user.email "wix_build_tools@wix.com"
```

Comment:
Also - what will happen if I run the tests locally - will it change my local git user email and name? Any way to do it using local env variable that will be set by the test?

Reply:
This is actually why it is advised to run within a container 😄 
I'll add a validation / read from sessioned git env vars to avoid tampering with local environment if testing suite executed locally

Reply:
Done.